### DISCUSSION TITLE
Connect Supabase to Looker Studio (Google Data Studio)?

### DISCUSSION THREAD

**Discussion by meibe-ab** [supabase/supabase#27448] [Questions]
Anyone done that and how have you done it? 

Thera are a few postgres connectors. I havn't gotten the free one to work and I suspect it is a version thing (it seems to need postgres 14). Thera are paid connectors that I don't want to pay for. And some that goes by API. Seems to be the easy route.. but it's kind of a jungle. 

My dream would be a connector that asks for my public supabase API key and uses that AND forces users to be logged in with their account (as i have it in supabase auth) and then use RLS based on that for the rows returned. 

I've started coding a custom one but it seems to be a bit trickey..

**Comment by embat-consultora** [comment]
were you able to connect supabase with looker? I'm having the same issue. Thanks!

**Reply by lkobus** [reply]
> were you able to connect supabase with looker? I'm having the same issue. Thanks!

yea i managed to connect, each data source you have to input the connection data, after that you can select a table or make a custom query, this will register a source in your current context (main, page...)

In the supabase in the connection the host is located after the password and before the port, the port is there, the password is the one you regenerate on the settings, db name and username will be show on connect (this connect option is on the top nav bar write connect)

**Reply by lkobus** [reply]
You can only query, i could not edit any data

**Reply by meibe-ab** [reply]
> were you able to connect supabase with looker? I'm having the same issue. Thanks!

I wanted a more integrated dashboard in my app, so I went with https://www.syncfusion.com/ . Long time goal is to also have a more powerfull dashboard in Looker Studio. Will probably end up writing (or try to vibe-code) my own connector.

**Comment by espeditodelgaudio-mercury** [comment]
In this case, supabase then will be slow?
